packages/@okta/i18n/src/properties/login.properties

```diff
@@ -915,6 +915,8 @@ oie.phone.invalid = Invalid Phone Number.
 oie.email.label = Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
+oie.email.verify.primaryButton = Send me an email
+oie.email.verify.subtitle = Verify with an email link or enter a code sent to
```

Comment:
this should include a placeholder for the email. Something like 
```
Verify with an email link or enter a code sent to <$1>{0}</$1>
```
That way you can add the html tags needed surrounding the email text

Note: also add a comment regarding what `{0}` means. Take a look one line below

Reply:
is the suggestion to add hbs template for this text?